docs/GettingStarted.md

```diff
@@ -64,7 +64,8 @@ of the Datadog tracer, you can override the following defaults:
       tracer: Datadog.tracer,
       debug: false,
       trace_agent_hostname: 'localhost',
-      trace_agent_port: 8126
+      trace_agent_port: 8126,
+      env: Rails.env
```

Comment:
So do you think that https://github.com/DataDog/dd-trace-rb/pull/101 will be the final approach?